### PR TITLE
-Add alt attribute for instagram img container

### PR DESCRIPTION
**Pull request by patHyatt**
This PR addresses a couple of accessibility issues that exist with the amp-instagram component.

Namely:
1) rendered  tag that houses the Instagram image does not have option for an alt attribute to make image screen reader friendly.
2) iframe pulled in from Instagram does not have a title attribute which helps screen reader users differentiate frames.

Issue #3503 details the amp-instagram component use in markup and rendered output.

*Commit log*
-Add title attribute for instagram iframe
-Update instagram tests to include alt and title attribute checks

### CODE REVIEW

File: extensions/amp-instagram/0.1/amp-instagram.js [hunk 1]
```diff
@@ -87,6 +88,10 @@ class AmpInstagram extends AMP.BaseElement {
     image.setAttribute('src', 'https://www.instagram.com/p/' +
         encodeURIComponent(this.shortcode_) + '/media/?size=l');
     image.setAttribute('layout', 'fill');
+
+    //Push alt to underlying amp-img component
```
Comment (mkhatib) [inline extensions/amp-instagram/0.1/amp-instagram.js]: No need for comment.

File: extensions/amp-instagram/0.1/amp-instagram.js [hunk 2]
```diff
@@ -110,6 +115,8 @@ class AmpInstagram extends AMP.BaseElement {
     this.iframe_ = iframe;
     iframe.setAttribute('frameborder', '0');
     iframe.setAttribute('allowtransparency', 'true');
+    //title frame for a11y
```
Comment (mkhatib) [inline extensions/amp-instagram/0.1/amp-instagram.js]: Nit: `// Add title to the iframe for better accessibility.` Or drop comment.

File: test/manual/amp-instagram.amp.html [hunk 3]
```diff
@@ -26,6 +26,7 @@
   <h1>amp #0</h1>
   <amp-instagram
     data-shortcode="fBwFP"
+    alt="Guy computing on a couch."
```
Comment (mkhatib) [inline test/manual/amp-instagram.amp.html]: Nit: `Malte Ubl computing on a couch.` 😄

Comment (erwinmombay) [inline test/manual/amp-instagram.amp.html]: 👍

Comment (erwinmombay) [inline test/manual/amp-instagram.amp.html]: better yet, use cramforce 🎆

**Comment by cramforce** [comment]
Great. This will also need a validator change.

**Comment by mkhatib** [comment]
@cramforce this doesn't need to wait for validator change right and can be merged?

**Comment by cramforce** [comment]
It can. But somebody needs to make the change!

**Comment by patHyatt** [comment]
I can add this, to the issue @mkhatib created #3533 later tonight/tomorrow.

**Comment by mkhatib** [comment]
@patHyatt mind actually adding the validator change to this same PR?

**Comment by Gregable** [comment]
Alternatively, I can also send a PR for this so that @patHyatt doesn't need to figure out our scheme for validation.

**Comment by patHyatt** [comment]
@Gregable Go for it, I won't be able to get back in to tihs for another 6+ hours.

**Comment by mkhatib** [comment]
Awesome thanks @Gregable 👍

**Pull request merged**